packages/maskbook/src/social-network-adaptor/twitter.com/collecting/post.ts

```diff
@@ -94,7 +93,8 @@ function registerPostCollectorInner(
                 ? `${isQuotedTweet ? 'QUOTED' : ''}${postIdParser(tweetNode)}${node.innerText.replace(/\s/gm, '')}`
                 : node.innerText
         })
-    startWatch(watcher, cancel)
+    watcher.startWatch(250)
```

Comment:
What does this mean?

Reply:
The most used watcher in our project is MutationObserverWatcher which is based on the MutationObserver. It no longer works on Twitter for unknown reason and we have other watchers, now I switch to IntervalWatcher which is based on setInterval.

250 is the interval of how often this watcher will check the state.

Reply:
How about other selectors? Do they need to use this watcher too?

Reply:
No idea. If they works just leave it for now.